src/vorta/views/diff_result.py

```diff
@@ -97,6 +98,8 @@ def __init__(self, archive_newer, archive_older, model: 'DiffTree'):
         self.archiveNameLabel_1.setText(f'{archive_newer.name}')
         self.archiveNameLabel_2.setText(f'{archive_older.name}')
 
+        diff_result_display_mode = SettingsModel.get(key='diff_files_display_mode').str_value
+        self.comboBoxDisplayMode.setCurrentIndex(int(diff_result_display_mode))
         self.comboBoxDisplayMode.currentIndexChanged.connect(self.change_display_mode)
```

Comment:
If you swap these lines, do you still need `get_diff_tree`?

Reply:
Didn't get you 😕 Could you please elaborate?

Reply:
lines 102 and 103. The signal will triggered by the programmatic change too.

Reply:
Ah, nice!